frontend/src/scenes/session-recordings/file-playback/SessionRecordingFilePlayback.tsx

```diff
@@ -51,26 +54,25 @@ export function SessionRecordingFilePlayback(): JSX.Element {
                     <SessionRecordingPlayer sessionRecordingId="" playerKey={playerKey} />
                 </div>
             ) : (
-                <Dragger
-                    name="file"
-                    multiple={false}
-                    accept=".json"
-                    showUploadList={false}
-                    beforeUpload={(file) => {
-                        loadFromFile(file)
-                        return false
-                    }}
+                <div
+                    ref={dropRef}
+                    className="w-full border rounded p-20 text-muted-alt flex flex-col items-center justify-center"
                 >
-                    <div className="p-20 flex flex-col items-center justify-center space-y-2 text-muted-alt">
-                        <p className="flex items-center gap-2 font-semibold">
-                            <IconUploadFile className="text-xl" />
-                            Load recording
-                        </p>
-                        <p className="text-muted-alt ">
-                            Drag and drop your exported recording here or click to open the file browser.
-                        </p>
-                    </div>
-                </Dragger>
+                    <LemonFileInput
+                        accept={'application/json'}
+                        multiple={false}
+                        onChange={(files) => loadFromFile(files[0])}
+                        alternativeDropTargetRef={dropRef}
+                        callToAction={
+                            <div className={'flex flex-col items-center justify-center space-y-2'}>
+                                <p className="flex items-center gap-2 font-semibold">
```

Comment:
There's some extra margin-bottom because of `p`'s default styling. Would look better as a `span` IMO